pkg/destroy/aws/aws.go

```diff
@@ -218,7 +218,7 @@ func (o *ClusterUninstaller) RunWithContext(ctx context.Context) ([]string, erro
 
 	// Delete the rest of the resources.
 	err = wait.PollImmediateUntil(
-		time.Second*10,
+		time.Minute*10,
```

Comment:
We do not want to change how often we attempt to delete resources. This will make the destroy experience significantly worse.